lefthook.yml

```diff
@@ -0,0 +1,14 @@
+pre-commit:
+    commands:
+        eslint:
+            glob: '*.{ts,tsx}'
+            run: yarn lint
+        ts-lint:
+            glob: '*.{ts,tsx}'
+            run: yarn tsc --noEmit
+        stylelint:
+            glob: '*.{css}'
```

Comment:
scss, do wr need here?

Reply:
I'm not sure, because we not going to setup scss, but we'll can discuss this issues